plugins/modules/ldap_search.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/python
```

Comment:
You seem to have included two copies of the same module in this PR.

The file in plugins/modules/ must be a **symlink** to the file in plugins/modules/net_tools/ldap.